logic/system.js

```diff
@@ -45,9 +45,11 @@ async function getBitcoinP2PConnectionDetails() {
     try {
         const address = await diskLogic.readBitcoinP2PHiddenService();
         const port = String(constants.BITCOIN_P2P_PORT);
+        const connectionString = `${address}:${port}`;        
```

Comment:
This seems redundant. Why do we need to do this server side?

Reply:
I wrote this a little while ago (2 months haha), so I assume it was to keep consistency with the other connection details objects. We can remove this if you think it's better.